Resources/Prototypes/Reagents/medicine.yml

```
@@ -164,13 +164,13 @@
         conditions:
         - !type:ReagentThreshold
           min: 30
-        probability: 0.02
+        probability: 0.02 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)